src/detect-engine-analyzer.c

```diff
@@ -859,6 +859,68 @@ void EngineAnalysisRules2(const DetectEngineCtx *de_ctx, const Signature *s)
     SCReturn;
 }
 
+typedef struct analyzer_item {
```

Comment:
style: UseThisStyleForStructNames

Reply:
> I like this approach. Much cleaner.

Great ... i'll move forward with this. This was a "POC"; I'll make sure your comments are addressed in the next pr.